TLM/TLM/UI/TrafficManagerTool.cs

```diff
@@ -948,16 +948,26 @@ public struct NodeVisitItem {
 				ExtCitizenInstance driverInst = ExtCitizenInstanceManager.Instance.ExtInstances[CustomPassengerCarAI.GetDriverInstanceId((ushort)i, ref Singleton<VehicleManager>.instance.m_vehicles.m_buffer[i])];
 				bool startNode = vState.currentStartNode;
 				ushort segmentId = vState.currentSegmentId;
-				ushort vehSpeed = SpeedLimitManager.Instance.VehicleToCustomSpeed(vehicle.GetLastFrameVelocity().magnitude);
 
+				// Some magical constant converting magnitudes into km/h
+				float vehSpeed = vehicle.GetLastFrameVelocity().magnitude * 5f;
```

Comment:
No need magic numbers...
```suggestion
				float vehSpeed = SpeedLimit.ToKmphPrecise(vehicle.GetLastFrameVelocity().magnitude / 8f);
```
Dividing by 8f will give you vehicle speed (in game speed units).